docs/general_usage.md

```diff
@@ -33,6 +33,45 @@ The CLI will attempt to load credentials in the following order of priority:
 1. From environment variables, if set
 1. From the active profile
 
+## Subaccounts
+
+Instructions on how to create a Twilio subaccount can be found in the [Twilio docs](https://support.twilio.com/hc/en-us/articles/360011348693-View-and-Create-New-Twilio-Subaccounts).
+
+### API Commands
+
+For non-`twilio api:core` commands, with your subaccount SID, please use profiles (via `twilio login` to create a new profile or `twilio profiles` to manage existing credentials).
+
+### API Core Commands
+
+You can’t use the token created by `twilio login` to access subaccounts as it creates a Standard key which is scoped for master account only. An appropriate API key can be created from a Master Account here: https://www.twilio.com/console/project/api-keys/create.
```

Comment:
Well, you _can_, it just has to be a subaccount profile. Wording should reflect something like "twilio-cli profiles created for master (parent?) accounts cannot be used to manage subaccounts. You can use the above method of creating a profile for the subaccount or the following:"